FhirLoader.Tests.E2E/UnitTest.cs

```diff
@@ -0,0 +1,83 @@
+using FhirLoader.Tool;
+
+namespace FhirLoader.Tests.E2E
+{
+    public class UnitTest
+    {
+        private const string fhirURL = "https://workpacesdkfhir-sjbuildfhir.fhir.azurehealthcareapis.com/";
```

Comment:
Best practice would be to load this from a setting file or environment variable

Reply:
added in secrete,json file.

{
  "FhirURL": "",
  "Batchsize": "500",
  "Concurrency": "50",
  "BlobBundlePath": "",
  "BlobndJsonPath": ""
}